CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.9.0] - 2021-08-17
```

Comment:
Thoughts on having this be `Unreleased` until we tag `main` ?

Reply:
@bspellacy not sure I follow. What's the release flow you have in mind? I'm planning on releasing all the SDKs then the changes to patch (docs)